libcaf_core/caf/blocking_actor.test.cpp

```diff
@@ -44,6 +48,18 @@ TEST("timeout_in_scoped_actor") {
   check(timeout_called);
 }
 
+TEST("spawn blocking actor") {
+  auto blocking_actor = sys.spawn(testee);
+  self->mail(42).send(blocking_actor);
+  dispatch_messages();
+  auto received = std::make_shared<bool>(false);
+  self->receive([received](int i) {
+    *received = true;
+    test::runnable::current().check_eq(i, 42);
```

Comment:
Should work if you just capture `this`

```suggestion
    check_eq(i, 42);
```